catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/NitfGmtiTransformer.java

```diff
@@ -108,27 +112,31 @@ private void transformTargetLocation(Metacard metacard) {
                 WKTReader wktReader = new WKTReader(geometryFactory);
                 Geometry geometry = wktReader.read(locationString);
 
-                LOGGER.debug("Setting the metacard attribute [{}, {}]", Core.LOCATION,
+                LOGGER.debug("Setting the metacard attribute [{}, {}]",
+                        Core.LOCATION,
                         geometry.toText());
                 metacard.setAttribute(new AttributeImpl(Core.LOCATION, geometry.toText()));
             }
 
         } catch (ParseException e) {
-            LOGGER.debug(e.getMessage(), e);
+            LOGGER.warn(e.getMessage(), e);
```

Comment:
I recommend squashing your commits so reviewers can look at the entirety of CAL-132 changes on a single page (and omit CAL-79)